control/controlclient/direct.go

```diff
@@ -915,6 +932,14 @@ func (c *Direct) sendMapRequest(ctx context.Context, maxPolls int, readOnly bool
 		} else {
 			vlogf("netmap: got new map")
 		}
+		if resp.ControlDialPlan != nil {
+			if c.dialPlan != nil {
+				c.logf("netmap: got new dial plan from control")
+				c.dialPlan.Store(resp.ControlDialPlan)
+			} else {
+				c.logf("netmap: new dial plan; nowhere to store it")
```

Comment:
if this is expected, it maybe should be [v1]. If it's unexpected, mark it with `[unexpected]` which is our convention used elsewhere for things that shouldn't happen but aren't fatal-worthy.

Reply:
Added `[unexpected]`